camera_rate: 60
